.travis/klee.sh

```diff
@@ -208,6 +208,20 @@ else
 
   set +e # We want to let all the tests run before we exit
   lit -v .
+
+  # If metaSMT is the only solver, then rerun tests with non-default metaSMT backends
+  if [ "X${SOLVERS}" == "XmetaSMT" ]; then
+    if [ "X${METASMT_DEFAULT}" != "Xbtor" ]; then
+      lit -v --param klee_opts=-metasmt-backend=btor --param kleaver_opts=-metasmt-backend=btor .
+    fi
+    if [ "X${METASMT_DEFAULT}" != "Xstp" ]; then  
+      lit -v --param klee_opts=-metasmt-backend=stp --param kleaver_opts=-metasmt-backend=stp .
+    fi
+    if [ "X${METASMT_DEFAULT}" != "Xz3" ]; then
+      lit -v --param klee_opts=-metasmt-backend=z3 --param kleaver_opts=-metasmt-backend=z3 .
+    fi
+  fi
+
```

Comment:
I think you misunderstood what I want. I do not want complicated logic. So in #549 I've removed it so you don't need to record the return codes of the test runs. The first failure will stop the testing.

For the CMake build it's really easy. The CMake build uses lit as well so just use the same command as you're using now. The only thing you need to change is that the path given to lit must relative to the root of the build directory (i.e. `test/` instead of `.`). You'll see in #549 I've change it so it is **always** the case that lit is invoked from the root of the build directory.